### PR TITLE
docs: Update local mode for live push

### PR DESCRIPTION
**Pull request by garethtdavies**
Refreshed local mode docs including a section for Livepush. 

All suggestions and feedback welcome!

Change-type: patch
Signed-off-by: Gareth Davies gareth@balena.io

### CODE REVIEW

**Comment by garethtdavies** [comment]
This is probably one for @shaunmulligan or @CameronDiver to review.

File: pages/learn/develop/local-mode.md [hunk 1]
```diff
@@ -1,146 +1,224 @@
...
+- In local mode, a device will not send logs back to the {{ $names.cloud.lower }} dashboard. Refer to the [local mode logs section](#local-mode-logs) to view logs in local mode.
+- Device and service environment variables set from the {{ $names.cloud.lower }} will not be applied to local mode containers. It is still possible to set environment variables in your `docker-compose.yml` or `Dockerfile`.
+- Changes to device [configuration variables][configuration], for example, `RESIN_HOST_CONFIG_gpu_mem`, will result in the device rebooting and applying those settings.
+- Actions such as _Restart_ and _Purge data_ from the {{ $names.cloud.lower }} dashboard will not apply to local mode containers.
+- When switching out of local mode and back to tracking releases from {{ $names.cloud.lower }}, the Supervisor will destroy any local mode containers and  volumes, as well as clean up unneeded base images, and then start up the application that {{ $names.cloud.lower }} instructs it to run.
 
-<img src="/img/local-mode/device-in-local-mode.png" width="100%">
+![Device in local mode](/img/local-mode/device-in-local-mode-20-01-09.png)
 
 ## Scan the network and find your device
```
Comment (CameronDiver) [inline pages/learn/develop/local-mode.md]: It's probably worth noting that this information can be found on the dashboard, with the ip address generally being correct for how you would access it. The <short uuid>.local should always work without a scan as well. This decreases the time for getting started

File: pages/learn/develop/local-mode.md [hunk 2]
```diff
@@ -1,146 +1,224 @@
...
+[Logs]    [1/9/2020, 1:47:03 PM] [main]
+[Live]    Device state settled
+[Logs]    [1/9/2020, 1:47:03 PM] [main] > simple-server-node@1.0.0 start /usr/src/app
+[Logs]    [1/9/2020, 1:47:03 PM] [main] > node server.js
+[Logs]    [1/9/2020, 1:47:03 PM] [main]
+[Logs]    [1/9/2020, 1:47:04 PM] [main] Example app listening on port  80
 ```
 
-These containers will have access to all the features and environment that balenaCloud deployed devices have. For example, you will still be able to query and use the [supervisor API][supervisor API] and the containers will be brought up automatically on boot.
+### Live Push
```
Comment (CameronDiver) [inline pages/learn/develop/local-mode.md]:
```suggestion
### Livepush
```

File: pages/learn/develop/local-mode.md [hunk 3]
```diff
@@ -1,146 +1,224 @@
...
+When detached, the services continue to run on the device, and you can access the logs using the `{{ $names.company.short }} logs` command, again passing the local IP address or `<short-uuid>.local`.
+
+```shell
+{{ $names.company.short }} logs 63ec46c.local
+```
+
+This command will output logs for the system and all running services. You may optionally filter the output to only include system or specific service logs using the available `--system` (`-S`) and `--service` (`-s`) options. For example, to only output the system logs:
+
+```bash
+{{ $names.company.short }} logs 63ec46c.local --service
```
Comment (CameronDiver) [inline pages/learn/develop/local-mode.md]:
```suggestion
{{ $names.company.short }} logs 63ec46c.local --service <service name>
```

**Pull request merged**